### PR TITLE
Added Fonts and Homepage

### PR DESCRIPTION
**Pull request by ejaz-ahamed**

### CODE REVIEW

File: lib/core/constants/products_constants/product_constants.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'product_constants.g.dart';
+
+class ProductConstants {
+  final addPdts = 'Add New Products';
```
Comment (thadathilsinan) [inline lib/core/constants/products_constants/product_constants.dart]: add prefix 'txt' with all text constants

**Review by thadathilsinan — CHANGES_REQUESTED** [review]

**Comment by ejaz-ahamed** [comment]
changed

File: lib/core/constants/products_constants/product_constants.dart (hunk 1 above)
Comment (ejaz-ahamed) [reply lib/core/constants/products_constants/product_constants.dart]: changed

File: lib/core/constants/products_constants/product_constants.dart [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'product_constants.g.dart';
+
+class ProductConstants {
+  final txtaddPdtTitle = 'Add New Products';
```
Comment (thadathilsinan) [inline lib/core/constants/products_constants/product_constants.dart]: change into camel case

**Review by thadathilsinan — APPROVED** [review]

**Review by thadathilsinan — APPROVED** [review]

**Pull request merged**